examples/draw_state.rs

```diff
@@ -33,11 +35,19 @@ fn main() {
 
     let (mut device, mut factory) = gfx_device_gl::create(|s|
         window.get_proc_address(s) as *const std::os::raw::c_void);
-    // create the main color/depth targets
+        
+    // Create the main color/depth targets.
     let draw_size = window.draw_size();
     let aa = samples as gfx::tex::NumSamples;
     let dim = (draw_size.width as u16, draw_size.height as u16, 1, aa.into());
-    let (output_color, output_stencil) = gfx_device_gl::create_main_targets(dim);
+    let color_format = <Srgba8 as Formatted>::get_format();
+    let depth_format = <DepthStencil as Formatted>::get_format();
+    let (output_color, output_stencil) =
+        gfx_device_gl::create_main_targets_raw(dim,
```

Comment:
`piston_window` has its own private `create_main_targets`. Not sure about whether `gfx_window_glutin` can be used. I think it would be messy to have backends just for gfx windows.